evidence/alexggh/0001-2024-10-21-rank-1-retention.md

```diff
@@ -0,0 +1,61 @@
+# Evidence-0001: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | 2024/10/21                                                                                  |
+| **Submitted by**| Alexandru Gheorghe                                                                          |
+
+
+## Member details
+
+- Matrix username: @alexggh:parity.io
+- Polkadot address: 14uA7Vc828e2Q6oL5GBHP9UzTkEvwqbroERwRmucGrLmPuuL
+- Current rank: 1
+- Date of initial induction: 2024/08/14
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: Parachains consensus
+
+
+## Reporting period
+
+- Start date: 2024/07/25
+- End date: 2024/10/21
+
+
+## Evidence
+During the reporting I was mostly focused on the scalability and stability of the Polkadot network.
```

Comment:
Quoting the advice from _ggwpez_ to all Rank 1 members submitting their evidence:
"I think instead of providing a list of PRs, it is better to only highlight the most impactful ones and explain why they are impactful. [...] Otherwise without context it is difficult to know what PRs of that list are actually impactful and which are not."

Reply:
I don't fully understand this comment,  I've seen this quote before writing the evidence and I tried do this exactly that by explaining the end goal and the steps taken to do that, with the lists of most impactful PR to achieve those steps.

Can you help with a few concrete examples where you think it does not follow the advice from ggwpez, thank you!

Reply:
Will do thank you!

Reply:
Let me know if it is better now, thank you!

Reply:
It does give a much detailed breakdown of your work! Cheers.